app/models/chargeback_rate.rb

```diff
@@ -150,7 +150,7 @@ def ensure_unassigned
   end
 
   def ensure_nondefault
-    if default?
+    if default? || description == 'Default Container Image Rate'
```

Comment:
@hstastna few questions, can't description be edited, or have same chargeback rate with same description, or is description unique among chargeback rates?

Reply:
@gtanzillo @lpichler if "Default Container Image Rate" is a default rate that gets seeded from yml, why dont we change yml to populate default field as true here https://github.com/ManageIQ/manageiq/blob/master/db/fixtures/chargeback_rates.yml#L155

Reply:
@h-kataria If I click on Edit this Chargeback rate and try to change the description of 'Default Container Image Rate' and click on Save, the flash message occurs:
`Can not change description of 'Default Container Image Rate'`
so I think this is ok.

Reply:
@hstastna i see that change was made here https://github.com/ManageIQ/manageiq-ui-classic/pull/269 to not allow edit of Default Container Image chargeback rate. I still think it should have been marked as default rate to begin with. @gtanzillo @lpichler @dclarizio any comments?

Reply:
@h-kataria we added it here https://github.com/ManageIQ/manageiq/pull/13063  and what I remember we needed to distinguish this specific rate and it was possible only according to name. But yes it isn't the reason why we didn't set  `ChargebackRate#default` to true in yaml. Maybe it was causing issues. Can you remember @zeari ?

Reply:
What we needed was a default-fallback rate, that if no rate is assigned that one is used.
We also needed the ability to edit it which wasnt possible for default rates.

Since default rates were not editable we made a special case of out "Default container image rate". The only place where that is used is in Chargeback for container images.